R/sim.cs-utils.r

```diff
@@ -101,3 +105,7 @@ mkbaseline <- function(kab,n=1, blims, ...){
   }
   return(yset);
 }
+
+#' @rdname mkbaseline
+#' @export
+mk_baseline <- mkbaseline
```

Comment:
- [ ] do the `lifecycle` process for this function
- [ ] don't export this function